not compiling

In the current git version it cannot compile when not using `AVX512`:
```
yafu/factor/avx-ecm/avx_ecm_main.c:649:(.text+0x1b98): undefined reference to `veckmul_redc'
/usr/bin/ld: yafu/factor/avx-ecm/avx_ecm_main.c:650:(.text+0x1baa): undefined reference to `vecksqr_redc'
```
Applying this [commit](https://github.com/gilcu3/yafu/commit/1ef62caed9f6c1b6548de37783f10dd64f6127c0) fixes the issue.

--- a/factor/avx-ecm/vecarith52_special.c
+++ b/factor/avx-ecm/vecarith52_special.c
@@ -11269,4 +11269,19 @@ void vectsqr_redc(vec_bignum_t* a, vec_bignum_t* c, vec_bignum_t* n, vec_bignum_
     vectmul_redc(a, a, c, n, s, mdata);
     return;
 }
+
+
+
+#else
+
+void veckmul_redc(vec_bignum_t* a, vec_bignum_t* b, vec_bignum_t* c, vec_bignum_t* n, vec_bignum_t* s, vec_monty_t* mdata)
+{
+    return;
+}
+
+void vecksqr_redc(vec_bignum_t* a, vec_bignum_t* c, vec_bignum_t* n, vec_bignum_t* s, vec_monty_t* mdata)
+{
+    return;
+}
+
 #endif

